README.md

```diff
@@ -4,6 +4,8 @@ JavaScript Model for Blockchain.info wallet.
 
 ## Build
 
+Install Yarn: `npm install -g yarn`
```

Comment:
This is not the preferred way, see: https://yarnpkg.com/en/docs/install#alternatives-tab. You could either just link to that or change it to `brew install yarn` (only for Mac).